src/components/AddRecipe.js

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import {Container, Row, Col, Button} from 'react-bootstrap';
+import {FaPlus} from 'react-icons/fa';
+
+ class AddRecipe extends React.Component {
+    state = {
+        recipetitle: "",
+        preptime: "",
+        cooktime: "",
+        serves: "",
+        measurement: [{measurement:""}],
+        unit:  [{unit:""}],
+        ingredient:  [{ingredient:""}],
+        steps: [{steps:""}],
+        number: [{number:""}],
+      };
+    
+      submitHandler = event => {
+        event.preventDefault();
+        event.target.className += " was-validated";
+      };
```

Comment:
The submitHandler does not send the data to the backend. And clicking the CreateRecipe button currently has no action/effect.

Reply:
This issue is still not fixed. Submitting the form has not functionality.